actors/builtin/miner/sectors.go

```diff
@@ -0,0 +1,89 @@
+package miner
+
+import (
+	"fmt"
+
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+
+	"github.com/ipfs/go-cid"
+)
+
+func LoadSectors(store adt.Store, root cid.Cid) (*Sectors, error) {
+	sectorsArr, err := adt.AsArray(store, root)
+	if err != nil {
+		return nil, err
+	}
+	return (*Sectors)(sectorsArr), nil
+}
+
+// Sectors is a helper type for accessing/modifying a miner's sectors. It's safe
+// to pass this object around as needed.
+type Sectors adt.Array
+
+func (sa *Sectors) array() *adt.Array {
+	return (*adt.Array)(sa)
+}
```

Comment:
Nit: I prefer unexported methods last, optimising for top-down understanding by the user over the implementer.